File: tasks/jetpack-compose/app/src/main/java/live/ditto/compose/tasks/data/Task.kt
package live.ditto.compose.tasks.data

import live.ditto.DittoDocument
import java.util.*

data class Task(
    val _id: String = UUID.randomUUID().toString(),
    val body: String,
    val isCompleted: Boolean
) {
    constructor(document: DittoDocument) : this(
        document["_id"].stringValue,
        document["body"].stringValue,
        document["isCompleted"].booleanValue
    ) {

    }
}

File: tasks/xamarin.ios/Task.cs
using System;
using System.Collections.Generic;
using DittoSDK;

namespace Tasks
{
    public class Task
    {
        public string _id;
        public string body;
        public bool isCompleted;

        public Task(DittoDocument document)
        {
            this._id = document["_id"].StringValue;
            this.body = document["body"].StringValue;
            this.isCompleted = document["isCompleted"].BooleanValue;
        }
    }
}
